src/app/Models/Setting.php

```diff
@@ -11,4 +11,31 @@ class Setting extends Model
 
     protected $table = 'settings';
     protected $fillable = ['value'];
+
+    public static function get($key)
+    {
+        $setting = new self();
+        $entry = $setting->where('key', $key)->first();
+
+        if (!$entry) {
+            return;
+        }
+
+        return $entry->value;
+    }
+
+    public static function set($key, $value = null)
+    {
+        $setting = new self();
+        $entry = $setting->where('key', $key)->first();
+
+        if (!$entry) {
```

Comment:
I wonder if this should actually do a `firstOrFail` and just throw an exception.

Or `firstOrUpdate()`.

Or if this should make an entry if `$key` cannot be found.

Reply:
True. ```firstOrFail()``` in https://github.com/Laravel-Backpack/Settings/commit/3064341962c9018df725ac17fb2defee24f7eef7